src/rulelib.cpp

```diff
@@ -306,20 +309,22 @@ ascii_to_vector(char *line, size_t len, int *nsamples, int *nones, VECTOR *ret)
 	}
 
 	/* Store val if it contains any bits. */
+    // Changed to make the last non-full v_entry left-aligned
```

Comment:
Why?

Reply:
Because every v_entry would be filled left-aligned, but in such a way that if the last v_entry was partially empty it would be right aligned (because the left-shift only happens as long as there are bits to assign). This ensures that rule_isset and rule_set work properly without GMP.